yasa-ui/src/style/iconfont/iconfont.css

```diff
@@ -0,0 +1,121 @@
r1h/t7FNt8HRxE5aVPE3fE79wcFnyC9dklIz12RdFROp8jkxa2tFHbr+me7vDXJU9B/wmT7aNLWsaHmrUbTyxQndFatpKnZWpPA2rCv+PQ0OyZWNtFBrZn/wIvnfvPV1KnAUufIgNTGcFJbYFPHSodHgIoRrhjAE+lWUttGh8HsA0u5dhJMBdDlezEFMv5ubtbn2o0tT6PMMiVpaNv2IUOH5DIvVRq/WwlRK6sKVibpC2n/H7RONKeYqaOgG340NPUVPgGeOBeILbEsrZbKzL9Z6klYpMh4ctjrctrEOm4oJdlq34RODQTUVgbF/iOsHtJ90a4TH/+u86/myOF3r/7BNn/Gpgupy8ao0klZYmIQfe2kd2cNJgOTowgC5nsE2KOpActGnLlib0QRna3o6WIdmVy8F18z7HGDlFnQsHgKtO6XWMDvYFv0pJFJ1Dt/LS1GSdaFoK7VI6Jh/g+hjJpbwlT5DfHsg98LdsgsIGJ8ujQlHhsETxrG1a1R6iACh+f/uWx/YdG5nr79za6HGdxbrGf2+ZbRexPIwRdpLN4DTULH5qQbxFsCNeJlgou2saOHTOSQ6DsyzCLtuSMEy0JQ5V2hZy6NwTVHuFNd1KC8Jloh3hFtHi6WKkYugr9xBsCXuPL07UiNGI5UMuaCoMPn9uRyr6++8rt3m/DIlXhFDPz4hRLgPJtT3zD+Ie8vpVeYg0I8n4MyN3Y6bh3xWlDOe0DUKDxWoZpmXFTFRCmt0Yma0T4Lgk/n3uw3mOUdTapUgkfmD629eZn5stU7V6WSbphPiTy+KsRHUFiJt7dYGO3bpBP0bVh0nD9jZxRU7E0vnRIXG1+KDIg4uoZ5bIiW62dLP8fK1GbUilYlvB+eOCctmV69v+yXRdGwOhhs7rPl/v4oQe+uQ4GzPj9bfGRrk4xi+IWVI6NdFcxsx0yZ1SFir1GFPLs72FK1sTHFyjPOjgD/IjMuuT/nplASL5VLn9+tJKH93xkjaeX9X8u2H48PmoDM2Ra6najZwEflSMeDY2gCCy/JLokmcnSAo7U5pXdR2Nr50o2yZStO4q/YV66Uc3i7bBrerbXrc9Bge9BsH70UMj9oHVYyzybpORmEmG0LJQY1Kj0FK4AfZgxZNEO1s2WNZvHNpdc+SaA4WjGyY1CoJh7W6cgNXMrYgOs6y3bBCPm7zuK7veyiysIKykEGmwJD3g4JoTxHH0pGQy0BF365CGCOmEIVJaCtI90tlWsZPmFeKJpUHzgCdhqcO2biUNkbqI4/V0XaQhopx0sqNLciXLKTN5axJEe/83k8lZe746dm/R0nPl/56tzamNAO+1Phd8Hlc99ozqdvs5T/qfGHk9qX5S5d3m01blGNVDCbA+U/gknb7ZvXtJjrobVoDctndzOuZLdQsW8PYm7/LIdTpMGzDK0V5lkk+pLccuXkoyDTr+tUo76FfSn1XBhfSkDkwqDnbRqDUrx4pNY4LNn8fQ5q+8V6h3NksbLzoXyCIssPmXSR1YcW+bXmI8fRk98h8mQWzTxKJ17KYvjfyUoe3GCCvKvlyNisxa422o5q36BoYLzV0zHPkGuvMZiy6Vesa8xnDlSU5B4QpyOT3zctzcR9pO2UiJeR6RX+H70SLHcOmX9ahUivqZ/GmYj0gTnPPEe24eK4wlZI2HjQvNWeJRIXOcpb8h1IcAlJb9+fPX7q39OzOzv/Z/S0t373qqz8APJRYGj+OF+HFt9c74iQnyYMbn1TrjTkP48fmONYlDjkEFIBb67iLs5uZGMj8aaO59ztWrzsecztVn+TkfP6Us67EJjdYMtVk6L1wSTK+Z55nIqm4yDVhu0v+a0YXM0lb1PvVWVokklbWptnYT6+BmVk3t+q1TwapF8zX6lPGRKh2z+3Nix57Vf8nkgPpH7Okf6Zjt27SZb+dP3aSO5n2/AqFV1Y457uFX4KfgcJqDEHBNZRRp1J16ZbNOtD0rBpk4YXri9Zd+0/5SkZ7R9+bznjj0vYsUcYI5/YVBhXVB3//Gp/dERsuzIO4nEhnoHks6EhJe0GEcbrRp87LmPocTpkIzodc0nPn+0zoGsu7TXIPN7jh0/OpL3x9amlg+9L1oJQnQ4VtR6dexyV2+QRhuHG2yyQzDIlHaF2wyjTIOF4k6/BJmwue78flu81vYcrm5N0MMmtXLrCyPhevY+fz5biBeOPsgmD0jGWMCAGD96LSjbrmMZaqqGWswAyp6E/2OyeOoCEP+y2sUO4q+BZj9IxpDWWPB6MdgtilqxSTZibXoZ3XafSwQ6pKjmBeq7ujArEzRteiA2jGGwayISHDQZoXZFuwh1ggzWefQk/ajKi0OQV8AzLZhPy1QNR+1YbQRyabAw9q6Nofu/4VA/uey7PYQwSDj/Y5uSqJAgJMsAQRgtBHwB70F8Nt0kWOIW5+ZdzV402lw8+wsUiH2gbcYwtkRKvv0gCRvABhZ3K4B91hA29oZgw1oJwyFOWDMMCCAlYFLcWJ4IIM7UwMCBDESOEJE92TQhGrAAMERAcABljoMAQbsZyjQYJRhwIAJA5dyl+FBB94wNWAgGCMBF9H0SYa7bD5Wb0pg5F33dBmlBDfPVtG+3xSvDJpOVwv+SVsuc4ftPl/7RYW0j3baLR7Nggsqvft0Xo2YxVWVM6Ftk1k97XaBaneL0g9WgzclsGBF73TPTTNKCXe2VeHXf1O8MqjhlJtN/0lbfnznYGtvYXyJYnXKUzm43eKRMRWcMNtbeueTSWJZJU6lGzsT2lZqoerJjl0q2Irt4QP9+yE8facVL3YNoWFg4eARQAjiP639LkOO4nA8nS/X2/3xfL0Ze8Ep9ZB53F3j5E08TRJxnXSAl+VL5EJUSafIcvWTyvAkG9I2Z4kxlzh7xF3kODnDDSZNWOe3J8Bj1p7NqF/kEr4TFKSJKdEUOEObK1XOCGsU5ofOLWNpU0ygtvTQUiegfoJHoZ/1UCDS0ueG0mfkORgAAAAA') format('woff2'),
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/yasa-org/yasa/pullRequest?prid=5815330)